sourced/ml/core/models/id_splitter.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import string
+from typing import List, Sequence, Tuple
+
+import keras
```

Comment:
We should use the Tensorflow implementation instead of the original one. https://www.tensorflow.org/guide/keras